packages/strapi-plugin-users-permissions/admin/src/translations/en.json

```diff
@@ -0,0 +1,113 @@
+
+  "Auth.form.register-success.email.label": "Email send with success to",
+  "Auth.form.register-success.email.placeholder": "mysuperemail@gmail.com",
+
+  "Auth.link.forgot-password": "Forgot your password?",
+  "Auth.link.ready": "Reasy to sign-in?",
+
+  "components.Input.error.password.noMatch": "Password don't match",
+
+  "Controller.input.label": "{label} ",
+
+  "EditForm.inputSelect.subscriptions.label": "Manage subscriptions quotas",
+  "EditForm.inputSelect.subscriptions.description": "Limit the number of subscriptions per IP per hour.",
+  "EditForm.inputSelect.durations.label": "Duration",
+  "EditForm.inputSelect.durations.description": "Number of hours during the user can't subscribe.",
+
+  "EditForm.inputToggle.label": "One account per email addres",
+  "EditForm.inputToggle.description": "Disallow the user to create multiple accounts using the same email address with different authentication providers.",
+
+  "EditPage.cancel": "Cancel",
+  "EditPage.submit": "Save",
+  "EditPage.form.roles": "Roles details",
+  "EditPage.form.roles.label.description": "Description",
+  "EditPage.form.roles.label.name": "Name",
+  "EditPage.form.roles.label.users": "Users associated with this role ({number})",
+  "EditPage.header.title": "{name} ",
+  "EditPage.header.title.create": "Create a new role",
+  "EditPage.header.description": "{description} ",
+  "EditPage.header.description.create": " ",
+
+  "EditPage.notification.permissions.error": "An error occurred while fetching permissions",
+  "EditPage.notification.role.error": "An error occurred while fetching the role",
+
+  "HeaderNav.link.advancedSettings": "Advanced settings",
+  "HeaderNav.link.emailTemplates": "Email templates",
+  "HeaderNav.link.providers": "Providers",
+  "HeaderNav.link.roles": "Roles",
+
+  "HomePage.header.title": "Auth & Permissions",
+  "HomePage.header.description": "Define the roles and permissions for every one of them",
+
+  "InputSearch.placeholder": "Search for a user",
+
+  "List.button.roles": "Add a New Role",
+  "List.button.providers": "Add a New Provider",
+
+  "List.title.emailTemplates.singular": "{number} email template is available",
+  "List.title.emailTemplates.plural": "{number} email templates are available",
+
+  "List.title.providers.disabled.singular": "{number} is disabled",
+  "List.title.providers.disabled.plural": "{number} are disabled",
+  "List.title.providers.enabled.singular": "{number} provider is enabled and",
+  "List.title.providers.enabled.plural": "{number} providers are enabled and",
+
+  "List.title.roles.singular": "{number} role is available",
+  "List.title.roles.plural": "{number} roles are available",
+
+  "notification.error.delete": "An error occured while trying to delete the item",
+  "notification.error.fetch": "An error occured while trying to fetch data",
+  "notification.success.delete": "The item has been deleted",
+
+  "plugin.description": "Protect your API with a full-authentication process",
```

Comment:
Why is the 'plugin' word in "lowercase" and on the next line "Camelcase"? We should definitely apply more strict rules in this part of our codebase because it becomes messy (linter?)...